source/blog/2008-06-13-php-net-style-api-documentation-lookups.md

```diff
@@ -8,7 +8,7 @@ permalink: /2008/06/13/php-net-style-api-documentation-lookups.html
 <p>
 
 I have been using php.net style API lookups ever since I can remember.
-It is very nice being able to simply go to http://php.net/array\_walk
+It is very nice being able to simply go to https://secure.php.net/array\_walk
```

Comment:
I think the original idea was that you can go to the definition of `array_walk()` just by typing "php.net/array_walk" which is indeed simple. However, typing "https://secure.php.net/array_walk" is not that simple (people usually don't do that). Would it make sense to reword the original statement and get rid of HTTP/HTTPS there entirely since it's irrelevant?

Reply:
This blog post is old and irrelevant now. We don't have the equivalent functionality on the website now.